content/_code-samples/freeze/py/check_freeze_status.py

```diff
@@ -0,0 +1,40 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models import AccountLines
+
+client = JsonRpcClient("https://xrplcluster.com")
+
+print("connected to mainnet")
+
+# issuer address
+issuer_addr = "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn"
+
+# target address to query for freeze status
+target_addr = "rUpy3eEg8rqjqfUoLeBnZkscbKbFsKXC3v"
+
+# token name
+token = "USD"
+
+# build account line query
```

Comment:
Similar comment to before (sorry for redundancy) - please remove comments which explain just what the code does and capitalize the first letter of each comment.